cmd/juju/controller/addmodel.go

```diff
@@ -56,25 +56,24 @@ type addModelCommand struct {
 const addModelHelpDoc = `
 Adding a model is typically done in order to run a specific workload.
 
-To add a model, you must specify a model name. 
-Model names can be duplicated across controllers but must be unique for
-any given controller. Model names may only contain lowercase letters,
-digits and hyphens, and may not start with a hyphen.
-
-To add a model, Juju requires a credential.
-If a controller has a credential for the user creating the model, 
-it will be used.
-If a controller does not have a credential, 
-this command allows to specify desired named credential with --credential option.
-This option stores provided named credential in the controller.
-
-Another way of ensuring that a named credential is stored in the controller, is
-by running update-credential command.
-
-Named credentials are defined in Juju via either 
-"juju add-credential" or "juju autoload-credentials" where
-actual names are in the form of either 
-"credential-name" or "credential-owner/credential-name". 
+To add a model, you must specify a model name. Model names can be duplicated 
```

Comment:
I just noticed the existing text about uniqueness, which is wrong. Can you please fix that? The name is unique *per user*, in a controller. So alice and bob can each have their own model called "secret".